src/python/setup.py

```diff
@@ -17,6 +17,16 @@
 NON_MINIMAL_LIST = ["tensorflow", "resampy"]
 
 
+# pkgs not needed for minimal pkg
+NON_MINIMAL_LIST = [
+    "coremltools",
+    "llvmlite",
```

Comment:
You should get `llvmlite` if you don't have `resampy`.

Reply:
good to know.